src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Options/GraphQLContentOptions.cs

```diff
@@ -159,7 +159,7 @@ internal bool ShouldSkip(ContentTypePartDefinition definition)
         internal bool ShouldSkip(Type fieldType, string fieldName)
         {
             return HiddenFields
-                .Any(x => x.FieldType == fieldType && x.FieldName == fieldName);
+                .Any(x => x.FieldType == fieldType && x.FieldName.Equals(fieldName, StringComparison.OrdinalIgnoreCase));
```

Comment:
Hmmm... isent graphql case sensitive?

Reply:
there was a reason for this i've totally forgotten :|